src/test/java/org/ccci/util/mail/EmailAddressTest.java

```diff
@@ -16,4 +26,68 @@ public void testValidation_noTrailingSpaces()
     {
         EmailAddress.valueOf("matt.drees@ccci.org ");
     }
+
+    @Test
+    public void sendEmail() throws MessagingException {
+        /*
+          WARNING: do not hard code values here.
+          Secure data should be added to MailMessage.java which is added in .getignore
+         */
+        String mailServer = MailProperties.getMailServer();

```

Comment:
This can't be resolved due to https://github.com/CruGlobal/util/pull/20/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R14.
You'll need to not exclude that if you intend to leave the reference.

Reply:
The point of putting this in a separate file was that it could be .gitignore'ed

If you are looking for a template to put in your local environment:

>package org.ccci.util.mail;
>
> public class MailProperties {
>     public static String mailServer = "";
>     public static String mailServerUsername = "";
>     public static String mailServerPassword = "";
>     public static String mailServerPort = "";
> 
>     public static EmailAddress toEmailAddress = new EmailAddress("");
>     public static EmailAddress fromEmailAddress = new EmailAddress("");
>     public static EmailAddress replyToEmailAddress = new EmailAddress("");
>     public static String emailSenderName = "";
> }
>

Reply:
This will get a build failure without including `MailProperties`, which isn't acceptable.

Reply:
Ooh, ya. Forgot about that.